apps/probe/src/components/probe-list/probe-list.tsx

```diff
@@ -23,6 +24,18 @@ import {
 } from "../../hooks/use-probes";
 import styles from "./probe-list.module.css";
 
+const getOsVariant = (dos: string) => {
+  if (!dos) return;
+
+  if (dos.includes("linux")) {
+    return "linux";
+  } else if (dos.includes("darwin")) {
+    return "darwin";
+  } else if (dos.includes("windows")) {
+    return "windows";
+  }
```

Comment:
The format for all the dos are pretty much the same so we can do some light parsing
```suggestion
return dos.split("-")[0]
```

Reply:
Good to know, ty ty